services/firebase/package.json

```diff
@@ -2,7 +2,9 @@
   "name": "functions",
   "scripts": {
     "lint": "tslint --project tsconfig.json",
-    "build": "tsc",
+    "build": "yarn clean && tsc && yarn copy-files",
+    "copy-files": "copyfiles -u 1 src/**/*.js src/**/*.mjml dist/",
```

Comment:
`src/**/*.mjml` is wrong.
mjml is not in `src` folder, it is in `templates` folder.

that's the reason why I get the error.

Reply:
@iiio2 this is not solved yet, I don't see any new commits